--- a/security/keys/encrypted-keys/encrypted.h
+++ b/security/keys/encrypted-keys/encrypted.h
@@ -18,16 +18,7 @@

-#if defined(CONFIG_SECURE_KEYS)
 extern struct key *request_secure_key(const char *secure_desc,
 				      const u8 **master_key,
 				      size_t *master_keylen);
-#else
-static inline struct key *request_secure_key(const char *secure_desc,
-					     const u8 **master_key,
-					     size_t *master_keylen)
-{
-	return ERR_PTR(-EOPNOTSUPP);
-}
-#endif

 #if ENCRYPTED_DEBUG
 static inline void dump_master_key(const u8 *master_key, size_t master_keylen)
